granary/microformats2.py

```diff
@@ -293,6 +295,17 @@ def json_to_object(mf2):
     'tags': [json_to_object(cat) for cat in props.get('category', [])],
   }
 
+  lat, lng = prop.get('latitude'), prop.get('longitude')
+  if lat and lng:
+    try:
+      lat, lng = float(lat), float(lng)
+      # 2 leading places for latitude, 3 for longitude
+      obj['position'] = '%0+10.6f%0+11.6f/' % (lat, lng)
```

Comment:
blerg sorry I hadn't looked closely enough. microformats2 is not a class, so it's not obvious how to call postprocess_activity without making the method static which I don't think is a good idea.

I propose just leaving out "position" and adding a TODO for now... :shrug: